fvm/serviceaccount.go

```diff
@@ -7,11 +7,13 @@ import (
 )
 
 const systemChunkTransactionTemplate = `
-import FlowServiceAccount from 0x%s
+import FlowEpoch from 0x%s
+
 transaction {
-  execute {
-    // TODO: replace with call to service account heartbeat
- 	log("pulse")
+  prepare(serviceAccount: AuthAccount) {
```

Comment:
I don't think there is a singer to this tx, so account wont be provided

Reply:
Hmm, what would be the best way to load something from the service account storage?

Reply:
change EN code add this signer I guess